docs/imm-otel.md

```diff
@@ -0,0 +1,101 @@
+
+| **Component** | **Name**        | **Description**                                                                                       |  
+|---------------| :---------------:  |-------------------------------------------------------------------------------------------------------|
+| Receiver      | `signal-fx`     | Component that sets the endpoint for receiving metrics data with the SignalFx metric data format.     |      
+| Processor     | `memory_limiter`, `batch`, and `queued_retry` are the recommended processors.         | Component that pre-processes data before it is exported.                                              |      
+| Exporter      | `signal-fx`     | Component that forwards data to SignalFx with the metric data format.                                 |
+
+  
+See [Configuration](https://opentelemetry.io/docs/collector/configuration/ "OpenTelemetry Collector Configuration") for more information about the OpenTelemetry Collector components.
+
+To deploy an OpenTelemetry Collector:
+
+1. Download the [latest release](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases "OpenTelemetry Collector contributions releases") of the OpenTelemetry Collector contributions version from GitHub.
+2. Create a configuration file that defines components for the OpenTelemetry Collector. A sample configuration file is provided below. See [examples](https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples “OTeL examples”) for an OpenTelemetry Collector demo.
+```
+# Save this file as collector.yaml.
+# Extensions are optional and are provided to monitor
+# the health of the OpenTelemetry Collector.  
+    extensions:
+      health_check: {}
+      pprof: {}
+      zpages: {}
+# The following section is used to collect the
+# OpenTelemetry Collector metrics.
+    receivers:
+      signalfx:
+# The SignalFx receiver accepts metrics in the SignalFx
+# proto format. This allows the collector to receive
+# metrics from the OpenTelemetry Collector.
+    processors:
+# Processors are not enabled by default; however, the following
+# processors are recommended.
+    batch:
+# The batch processor must be defined in the pipeline after the
+# memory_limiter as well as any sampling processors.
+    memory_limiter:
+      ballast_size_mib: 683
+      check_interval: 2s
+      limit_mib: 1800
+      spike_limit_mib: 500
+ # Enabling the memory_limiter is strongly recommended for every pipeline. 
+    exporters:
+# One or more exporters must be configured.
+# Metrics
+     signalfx:
+        access_token: "YOUR_ACCESS_TOKEN"
+        realm: "YOUR_SIGNALFX_REALM"
+     service:
+       pipelines:
+         metrics:
+           receivers: [signalfx]
+           processors: [memory_limiter, batch]
+           exporters: [signalfx]
+       extensions: [health_check, zpages, pprof]
+```
+3. Deploy the Open Telemetry Collector.
+```
+otelcontribcol --config collector.yaml --mem-ballast-size-mib=683
```

Comment:
The ballast depends on the amount of memory allocated to the collector -- need some note on this

Reply:
Where can I find this information?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/tree/master/processor/memorylimiter